modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -2345,4 +2350,20 @@ private static void handleExceptionOnCleanUpTablesResources(
             nodeStoppingEx.set(true);
         }
     }
+
+    private static Collection<UUID> collectTableIndexes(TablesView tablesConfig, UUID tableId) {
```

Comment:
Another little nitpick: this method does not have to be `static`, since `tablesConfig` is a field of this class and can be removed from parameters

Reply:
Fix it.